docs/DYNAMIC_EXCHANGE_MANAGEMENT.md

```
@@ -0,0 +1,296 @@
+- ✅ **Dynamic**: Add/remove exchanges without restarting services
+
+## Usage
+
+### List Configured Exchanges
+
+```bash
+neuratrade exchanges list
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)